deployability/modules/workflow_engine/tests/data/wf-ko-no-path-on-do.yaml

```diff
@@ -0,0 +1,169 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+version: 0.1
+description: This workflow is used to test agents deployment with a single manager.
+variables:
+  agents-os:
+    - linux-ubuntu-22.04-amd64
+  manager-os: linux-ubuntu-22.04-amd64
+  infra-provider: vagrant
+  working-dir: /tmp/dtt1
+
+tasks:
+  # Generic agent test task
+  - task: "run-agent-tests-{agent}"
+    description: "Run tests uninstall for the {agent} agent."
+    do:
+      this: process
+      with:
+        args:
+          - modules/testing/main.py
+          - inventory: "{working-dir}/agent-{agent}/inventory.yaml"
+          - dependencies:
+            - manager: "{working-dir}/manager-{manager-os}/inventory.yaml"
+            - agent: "{working-dir}/agent-{agent}/inventory.yaml"
+          - tests: "install,register,stop"
+          - component: "agent"
+          - wazuh-version: "4.7.1"
+          - wazuh-revision: "40709"
+    depends-on:
+      - "provision-install-{agent}"
+      - "provision-manager"
+    foreach:
+      - variable: agents-os
+        as: agent
+
+  # Generic agent test task
+  - task: "run-agent-tests-uninstall-{agent}"
+    description: "Run tests uninstall for the {agent} agent."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/testing/main.py
+          - inventory: "{working-dir}/agent-{agent}/inventory.yaml"
+          - dependencies:
+            - manager: "{working-dir}/manager-{manager-os}/inventory.yaml"
+          - tests: "uninstall"
+          - component: "agent"
+          - wazuh-version: "4.7.1"
+          - wazuh-revision: "40709"
+    depends-on:
+      - "run-agent-tests-{agent}"
+      - "provision-uninstall-{agent}"
+    foreach:
+      - variable: agents-os
+        as: agent
+
+  # Unique manager provision task
+  - task: "provision-manager"
+    description: "Provision the manager."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/provision/main.py
+          - inventory-manager: "{working-dir}/manager-{manager-os}/inventory.yaml"
+          - install:
+            - component: wazuh-manager
+              type: aio
```

Comment:
```suggestion
              type: assistant
```

Reply:
Done in [6311d92](https://github.com/wazuh/wazuh-qa/pull/5190/commits/6311d92724994e505494d41ca00dd7d9cddda76f)